lib/errors/expiredPassword.js

```diff
@@ -0,0 +1,13 @@
+module.exports = errors => {
+  class ExpiredPasswordError extends errors.KuzzleError {
+    constructor() {
+      super('Expired password', 403, 'kuzzle-plugin-auth-passport-local.expired-password', 0x0ff001001);
```

Comment:
If you want to have an error ID and code, then you should declare these errors in the plugin's manifest, and throw them using the error manager instead (see https://docs.kuzzle.io/core/2/plugins/plugin-context/errors-manager/ )

If you don't care about that, you can leave the id and code undefined.

Reply:
Oh, I had missed this very nice feature..
I'll update it.

Reply:
done